CHARTER.md

```diff
@@ -0,0 +1,107 @@
+
+#### 1. Mission and Scope of the Technical Initiative
+
+- a. The mission of the Technical Initiative is to identify critical open source software (OSS) projects and secure those projects. This space allows members to collaborate together on these goals.
+
+- b. The scope of the Technical Initiative includes collaborative development under the Technical Initiative License (as defined herein) supporting the mission, including organizing collaboration activities, defining best practices, documentation, testing, integration, and the creation of other artifacts that support the mission.
+
+#### 2. Technical Steering Committee
+
+- a. The Technical Steering Committee (the "TSC") will be responsible for all oversight of the Technical Initiative. 
+
+- b. The TSC voting members are initially the Technical Initiative’s Maintainers. The Maintainers will be documented in the Technical Initiative repository. The TSC is responsible for determining the future process for defining voting members of the TSC, and any such alternative approach will also be documented appropriately.  Any meetings of the Technical Steering Committee are intended to be open to the public, and can be conducted electronically, via teleconference, or in person. 
+
+- c. The Technical Initiative generally will involve Collaborators and Contributors. The TSC may adopt or modify additional roles so long as the roles are documented in the Technical Initiative’s repository. Unless otherwise documented: 
+
+   - i. Contributors include anyone in the technical community that contributes effort, ideas, code, documentation, or other artifacts to the Technical Initiative; 
+
+   - ii. Collaborators are Contributors who have earned the ability to modify ("commit") text, source code, documentation or other artifacts in the Technical Initiative’s repository or direct the agenda or working activities of the Technical Initiative; and
+
+   - iii. A Contributor may become a Collaborator by a majority approval of the existing Collaborators. A Collaborator may be removed by a majority approval of the other existing Collaborators.
+
+   - iv. Maintainers are the initial Collaborators defined at the creation of the Technical Initiative. The Maintainers will determine the process for selecting future Maintainers. A Maintainer may be removed by two-thirds approval of the other existing Maintainers, or a majority of the other existing Collaborators.
+
+- d. Participation in the Technical Initiative through becoming a Contributor, Collaborator, or Maintainer is open to anyone, whether a OpenSSF member or not, so long as they abide by the terms of this Technical Charter. 
```

Comment:
I don't believe these fit best here as they are about initiative membership and not about the TSC.  Recommend moving the section 1  as c.   Membership

Reply:
May need you to explain more here

Reply:
Section 2 is all about the technical steering committee, these items (c and d) don't talk about the technical steering committee (other than their ability to adopt or modify those roles) rather they talk about membership and the members of the technical initiative.  If this is moved up to section 1 under the technical initiative area to discuss membership, you can call out the TSC defines these roles and may modify them in accordance with their process or needs of the community.